src/mapping/RadialBasisFctMapping.hpp

```diff
@@ -384,7 +384,7 @@ void RadialBasisFctMapping<SOLVER_T, Args...>::mapConsistent(const time::Sample
       for (int i = 0; i < this->input()->getGlobalNumberOfVertices(); i++) {
         in[i] = inputValues[i * valueDim + dim];
       }
-
+      PRECICE_INFO("LOOCV (no polynomial) for vector component {}: {}", dim, _rbfSolver->computeRippaLOOCVerror(in));
```

Comment:
Could we include the data name here?

Reply:
Mapping has no notion of the data anymore.

This info statement gives logs of each map invocation necessary context:
https://github.com/precice/precice/blob/11f3a544ac53f71adf0951df595f02b52ac58e75/src/precice/impl/DataContext.cpp#L135-L137